sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizePiiEntityAsyncClient.java

```diff
@@ -205,6 +146,31 @@ private Response<RecognizePiiEntitiesResultCollection> toRecognizePiiEntitiesRes
     private Mono<Response<RecognizePiiEntitiesResultCollection>> getRecognizePiiEntitiesResponse(
         Iterable<TextDocumentInput> documents, RecognizePiiEntitiesOptions options, Context context) {
         options = options == null ? new RecognizePiiEntitiesOptions() : options;
+
+        if (languageSyncApiService != null) {
+            PiiTaskParameters piiTaskParameters = new PiiTaskParameters();
+            piiTaskParameters = (PiiTaskParameters) piiTaskParameters
```

Comment:
looks like some parameters are missing, like `domainFiler`, `categoriesFilter`, `stringIndexType`

Reply:
This happens in other endpoints too